psi4/driver/driver_cbs.py

```diff
@@ -171,9 +175,16 @@ def _contract_bracketed_basis(basisarray: List):
         return basisstring
 
 
-def xtpl_highest_1(functionname: str, zHI: int, valueHI: float, verbose: bool = True, **kwargs):
-    r"""Scheme for total or correlation energies with a single basis or the highest
-    zeta-level among an array of bases. Used by :py:func:`~psi4.cbs`.
+### GENERIC FUNCTIONS
+def xtpl_highest_1(
+    functionname: str,
+    zHI: int,
+    valueHI: Union[float, core.Matrix, core.Vector],
```

Comment:
Optional: You can define `Extrapolatable = Union[float, core.Matrix, core.Vector]` once and then just use `valueHI: Extrapolatable` in this case. The advantages are that you save typing, it's clearer what these variable means, and allows you to change the type _en masse_ if need be.